### DISCUSSION TITLE
Problem with Switch controllers

### DISCUSSION THREAD

**Discussion by MagnusHydra** [darthcloud/BlueRetro#493] [Q&A]
#468
#146
Both seem to state the problem I am having. Using 4 switch PowerA Gamecube controllers.
If this is fixed by software how will we update the gamecube blueretro internal adapter?

**Comment by darthcloud** [comment]
Update process is documented here:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#5---ota-fw-update-page

**Comment by andrew171717** [comment]
I'm seeing the same with 1 of my Switch PowerA Gamecube controllers.

I have 2 models, the older one (model 1507451-01) has heavy laggy while the newer one (model 1507452-01) seems to work just fine (no lag).

Edit: I read your first comment on #146, so it's something to do with multiple switch controllers and there's a somewhat repeatable workaround. Gotcha, sorry for the extra comment.

**Reply by darthcloud** [reply]
There is various issue with Switch controller, mostly for multiplayer, but some do have issue for single player too.
Personally 1P is fine for me but I do have the multiplayer issue.

It's something I plan to investigate in the xmas break.

For now I can only recommend to try to use PS4 or PS5 as they are the most reliable one until I figure out the issue with Switch controllers.
